doc/source/user/absolute-beginners-outline.rst

```diff
@@ -0,0 +1,137 @@
+**************
+NumPy: The Absolute Basics for Beginners
+**************
+
+This is a working outline for a future section introducing NumPy to absolute beginners. If you have comments or suggestions, please don’t hesitate to reach out!
+
+
+
+- How to install NumPy
+  
+  - various operating systems 
+
+- How to import NumPy
+
+- What is an array
+  
+  - What else might it be called
+  
+  - What is its shape
+
+  - Can you reshape it?
+
+  - What’s the difference between a Python List and a NumPy array? Why use NumPy?
+
+  - What are the attributes of an array?
+
+  - Broadcasting example
+
+- How to create an array (ndarray object)
+- How to create a basic array
+- How to create an array from existing data
```

Comment:
One thing to be pragmatic about: reading CSV files. Everyone needs to deal with that at some point, usually sooner than later, and NumPy's functionality for it kind of sucks. So best to say that and recommend this as the most robust way to get an array out:
```
import pandas as pd

# if all columns are the same type
x = pd.read_csv('filename.csv').values

# otherwise, select the columns you need
x = pd.read_csv('filename.csv', columns=['float_colname_1', ...]).values
```

Reply:
Perfect!